pandas/tests/extension/test_arrow.py

```diff
@@ -2456,7 +2456,7 @@ def test_dt_properties(prop, expected):
                 hour=3,
                 minute=4,
                 second=7,
-                microsecond=5,
+                microsecond=2000,
```

Comment:
Could you also add a separate test where microseconds should return 0?

Reply:
Sure